src/VocabGamePlay.scss

```diff
@@ -1,26 +1,29 @@
 @import './shared.scss';
 
-* {
-  border: 1px dashed red;
-}
+// * {
+//   border: 1px dashed red;
+// }
 
 * {
   margin: 0;
   padding: 0;
   box-sizing: border-box;
+  color: $main-text;
 }
 
-#root {
+.vocabGamePlay--container {
   min-height: 100vh;
   display: flex;
   flex-direction: column;
+  background: $background-gradient;
 }
```

Comment:
it's not a personal preference! it looks ridiculous stretched wide lol. i didn't consider this because it looks fine at full width on my smaller monitor, thanks!